src/DynamoCoreWpf/ViewModels/Core/AnnotationViewModel.cs

```diff
@@ -536,11 +547,15 @@ private bool CanChangeFontSize(object obj)
         public AnnotationViewModel(WorkspaceViewModel workspaceViewModel, AnnotationModel model)
         {
             annotationModel = model;
+            var modelBase = (ModelBase)model;
```

Comment:
why do you need to cast it to ModelBase ?

Reply:
Removed